app/navigation/routeDetails/OfficeOfTransitCountryNavigator.scala

```diff
@@ -36,7 +35,7 @@ trait OfficeOfTransitCountryNavigatorProvider {
 
 class OfficeOfTransitCountryNavigator(
   index: Index
-) extends UserAnswersNavigator[OfficeOfTransitCountryDomain, RouteDetailsDomain]()(
+) extends UserAnswersNavigator[OfficeOfTransitCountryDomain, TransitDomain]()(
       OfficeOfTransitCountryDomain.userAnswersReader(index),
-      RouteDetailsDomain.userAnswersReader
+      TransitDomain.userAnswersReader
```

Comment:
The domain model that drives the "CheckMode" navigation is something we may need to come back to. The CountryOfRoutingNavigator uses the RouteDetailsDomain but it will probably be clearer when the entire journey is built which works better.

Reply:
Done